tests/drivers/fuel_gauge/sbs_gauge/src/test_sbs_gauge.c

```diff
@@ -344,5 +348,29 @@ ZTEST_USER_F(sbs_gauge_new_api, test_get_buffer_props__returns_ok)
 	zassert_ok(ret);
 }
 
+ZTEST_USER_F(sbs_gauge_new_api, test_charging_5v_3a)
+{
+	/* Validate what props are supported by the driver */
+
+	struct fuel_gauge_get_property props[] = {
+		{
+			.property_type = FUEL_GAUGE_VOLTAGE,
+		},
+		{
+			.property_type = FUEL_GAUGE_CURRENT,
+		},
+	};
+
+	zassume_ok(emul_fuel_gauge_set_battery_charging(fixture->sbs_fuel_gauge, 5000, 3000));
```

Comment:
Might consider making 5000 and 3000 symbols, since they're referenced more than once